### PR TITLE
Make link to learning path more friendly in innersource

### LINKED ISSUE #29: Learning Path link not GHES Friendly
## Bug Report

**Current behavior**
The course links to a learning path on GitHub.com and not to the learning path on GHES when deployed on GHES.

**Reproduction**
Steps to reproduce the behavior in the course:
1. Finish the class successfully.
2. Click on 'Innersource learning path'
3. It opens up this site: https://lab.github.com/githubtraining/paths/innersource:-theory-to-practice

**Expected behavior**
I would expect that LL deployed on GHES would link to the learning paths in GHES and not to the ones on GitHub.com

**Possible solution**
Not sure if you use the same containers for GHES vs. GitHub.com, but it would be nice to have that link correct when deployed out to GHES if possible.

**Additional context**
I should be able to work around this locally.  Pinged @crichID via Slack and requested someone also open up an issue against LL for including Learning Paths as a part of the deployment.

/cc @crichID @brianamarie @hectorsector @JasonEtco

### PR DESCRIPTION
**Pull request by brianamarie**
This resolves #29 by using [variables](https://lab.github.com/docs/writing-responses) instead of a hard-coded link to the InnerSource path.

I am going to deploy this branch and test, verifying it still works on GitHub.com. @amyschoen is there a way you could test this solution on GHE while this change is still on a branch?

### CODE REVIEW

**Comment by brianamarie** [comment]
Bummer - this is close, but not quite right. The `{{ owner }}` variable is referring to the owner of the repo (so the user) not the owner of the course. 

@JasonEtco @githubtraining/learning-engineering Is there a [variable](https://lab.github.com/docs/writing-responses) for the owner of the course? The use case is to automatically link to learning paths on GHE the same way we do on .com.

**Comment by amyschoen** [comment]
@brianamarie `{{ owner }}` worked fine for me in GHES.  Is this a case of it worked because I was the one taking the class and set up LL and it wouldn't for a different user?

**Comment by brianamarie** [comment]
> @brianamarie `{{ owner }}` worked fine for me in GHES. Is this a case of it worked because I was the one taking the class and set up LL and it wouldn't for a different user?

Yes, I believe so.

**Comment by brianamarie** [comment]
Because there is no way to gather the registration method from the payload before this comment, there's no way to do this using variables with the current structure. I will open an issue for Learning Lab to create a variable that is similar to `repoUrl`, but maybe `orgUrl` to reference the organization URL for a given course or path, regardless of who that owner is and where the instance of Learning Lab is.  

In the meantime, I believe this is blocked. 😞

**Comment by amyschoen** [comment]
That makes sense.  Thanks for looking into this.

**Comment by brianamarie** [comment]
@JasonEtco for the win - it looks like `{{ course.Owner.login }}` will work. I still need to test this, but I am optimistic!

File: responses/14_congratulations.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ You finished this course. You can see a deployed version of our work together at
 
 I won't track any further events in this repository.
 
-Your next steps are in the [Innersource learning path]({{ learningPathURL }}).
\ No newline at end of file
+Your next steps are in the [Innersource learning path]({{ GHE_HOST }}/{{ course.Owner.login }}/paths/innersource:-theory-to-practice).
```
Comment (JasonEtco) [inline responses/14_congratulations.md:17]: You can use `GITHUB_URL` here for a consistent setup on lab.github.com and on GHES!

```suggestion
Your next steps are in the [Innersource learning path]({{ GITHUB_URL }}/{{ course.Owner.login }}/paths/innersource:-theory-to-practice).
```

**Comment by hectorsector** [comment]
@brianamarie for what it's worth, lately I've been hardcoding dotcom-specific URLs for this kind of thing because using the variable assumes the GHE installation will have an up-to-date copy of the paths we expect. Using the Dotcom-specific URLs ensures that the learner will not get a 404.

**Comment by amyschoen** [comment]
@hectorsector, the Dotcom-specific URLs don't work because these repos also get bundled for the version that gets deployed on-prem.  It leads to a lot of supportability challenges because you have to redo all the GHES-specific changes every time the app is upgraded locally.  You're likely undoing a lot of what I did for PRs after getting this working onsite.

**Comment by hectorsector** [comment]
It's a good point, @amyschoen. I'll open an issue in the Learning Lab repo about how to handle cases like this nicely. Ideally the author would link to a course by title or some other identifier, and the app should resolve if that course exists in GHES and, if not, failsafe to Dotcom.

**Comment by brianamarie** [comment]
Given @jasonetco's vote of confidence, I'm going to go ahead and merge. @amyschoen, please let us know if this doesn't work. I will also move forward next steps to get this handy link added to the docs, so others can find it easily for similar situations in the future.

**Pull request merged**